NebulaNetwork/NebulaNetwork.csproj

```diff
@@ -3,10 +3,17 @@
   <ItemGroup>
     <Compile Remove="PacketProcessors\Session\LobbyUpdateValues.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+  </ItemGroup>
```

Comment:
We have https://github.com/hubastard/nebula/blob/master/NebulaModel/Utils/SimpleJsonParser.cs already so either that should be used or that should be removed in favor of including Newtonsoft.Json in Directory.Build.props.

Reply:
Ah was looking for that, however could not find it. I will make the change :)

Reply:
I made the requested changes :)